app/src/main/res/layout/activity_maps.xml

```diff
@@ -32,13 +32,29 @@
         app:srcCompat="@android:drawable/ic_dialog_map"
         tools:ignore="SpeakableTextPresentCheck" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
The button is displayed on top of the navigation bar
![image](https://user-images.githubusercontent.com/31632552/162175903-66c0d0fc-7f71-4f18-98b9-1dc3b4ce400a.png)

Reply:
it should be nice now